vyper.asciidoc

```diff
@@ -78,7 +78,7 @@ modifier onlyBy(address _account)
 
 By and large, the usual use case for a modifier, is to perform single checks before a function's execution. Given that this is the case, Vyper's recommendation is to do away with modifiers altogether and simply use in-line checks and asserts as part of a function. Doing this will improve audit-ability and readability, due to the fact that the Vyper function will follow a logical in-line sequence in plain sight, rather than having to reference modifier code which has been written elsewhere.
 
-Class inheritance:: TODO explain class inheritance and then paraphrase the following ... "requires people to jump between multiple files to understand what a program is doing, and understand the rules of precedence in case of conflicts (which class's function X is the one that's actually used?). Hence, it makes code too complicated to understand."
+Class inheritance:: Inheritance allows programmers to harness pre-written code by acquiring pre-existing functionality, properties and behaviors from existing software libraries. Inheritance is powerful and promotes the reuse of code. However, it also creates an issue for those who write and audit code. Inheritance forces programmers and code auditors, who seek to understand precisely what a program is doing, to jump between multiple files while storing complex information in their memory. It can be argued that, given the immutability of blockchains, leveraging powerful, obscure and sometimes unintelligible software libraries can wreak havoc in smart contract implementation. For example a programmer inadvertently triggering pre-written code in an incorrectly coded smart contract.
```

Comment:
This point belongs to the principles on which Vyper is based, which on the other hand, **currently could be subject of  debate** in the Ethereum community.

Therefore, IMHO, I am not sure that the book is the most appropriate place to **emphasize** it. I would only mention it **without emphasizing it too much**.

Anyway, let's wait @aantonop express his opinion and decide.